nets/DevRelaySpec.ts

```diff
@@ -39,15 +39,16 @@ export class DevRelaySpec extends DevNetSpec {
     return createCustomChainSpec(tempDir, binary, this.chain, (chainSpec) => {
       const genesisConfig = getGenesisConfig(chainSpec)
       if (parachainInfo.length) {
-        genesisConfig.paras.paras.push(
-          ...parachainInfo.map((
-            { id, genesis },
-          ): GenesisConfig["paras"]["paras"][number] => [id, [...genesis, true]]),
+        genesisConfig.paras?.paras.push(
+          ...parachainInfo.map(({ id, genesis: [state, wasm] }) =>
```

Comment:
- Can we add an explicit return type and remove the `as NonNullable<GenesisConfig["paras"]>["paras"][number]`?
- Can we expose this type as named from within `DevNetSpec`? `ChainSpecGenesisConfigParasEntry` perhaps?

Reply:
You could just write `[id, [state, wasm, true]] satisfies Narrow`